snake.js

```diff
@@ -1,56 +1,46 @@
-//_ Hi again, Emil!
-//_ Make sure you send me a mail when you read this so I know you read the code ;)
-
-
-
-//_ This is the state representing, below you'll find some simple functions that can be
-//_ used to display the state. An empty String ( '') is used when nothing is there.
-//_ 'S' is used to represent the snake body. 'H' is the head of the snake.
-//_ First we'll figure out how the snake moves and rules and so on.
-//_ Then we'll figure out how to represent the state better
-var state = [
```

Comment:
So no more state, we track the position of each part of the snake instead of rendering it directly...